test_scripts/TheSameApp/Policies/AppId/005_PTU_is_not_triggered_after_registration_app_with_different_app_id_during_timeout.lua

```diff
@@ -0,0 +1,90 @@
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3136
+--
+-- Description: Check that SDL does not trigger 2nd PTU for App2 from Mobile №2 after
+-- the first PTU for App1 from Mobile №1 is finished in case App1 and App2 are registered
+-- within 'ApplicationListUpdateTimeout'
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) Mobile №1 and №2 are connected to SDL and are consented
+-- 3) App1 is registered from Mobile №1 and triggers PTU
+--
+-- Steps:
+-- 1) App2 is registered from Mobile №2 during PTU for App1 from Mobile №1 is in progress
+--   Check: SDL does not send SDL.OnStatusUpdate and BC.PolicyUpdate to HMI during the app registration
+-- 2) First PTU is performed successful
+--   Check: SDL triggers new PTU after the first one is finished
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/TheSameApp/commonTheSameApp')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Data ]]
+local devices = {
+  [1] = { host = "1.0.0.1", port = config.mobilePort },
+  [2] = { host = "192.168.100.199", port = config.mobilePort }
+}
+
+local appParams = {
+  [1] = {
+    appName = "Test Appl",
+    isMediaApplication = true,
+    appHMIType = { "DEFAULT" },
+    appID = "0009",
+    fullAppID = "0000009"
+  },
+  [2] = {
+    appName = "Test Appl",
+    isMediaApplication = true,
+    appHMIType = { "DEFAULT" },
+    appID = "0010",
+    fullAppID = "0000010"
+  }
+}
+
+local function registerApp1()
+  common.hmi.getConnection():ExpectRequest("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" })
+  common.registerAppEx(1, appParams[1], 1, false)
+end
+
+local function registerApp2()
+  common.registerAppEx(2, appParams[2], 2, true)
+  if common.extendedPolicyOption == "EXTERNAL_PROPRIETARY" then
+    common.hmi.getConnection():ExpectRequest("SDL.OnStatusUpdate"):Times(0)
+  else
+    common.hmi.getConnection():ExpectRequest("SDL.OnStatusUpdate", { status = "UPDATING" })
+  end
+end
+
+local function secondPTUIsNotTriggered()
+  local expTable
+  if common.extendedPolicyOption == "EXTERNAL_PROPRIETARY" then
+    expTable = { { status = "UPDATING" }, { status = "UP_TO_DATE" } }
+  else
+    expTable = { { status = "UP_TO_DATE" } }
+  end
+  common.isPTUNotStarted()
+  common.hmi.getConnection():ExpectRequest("SDL.OnStatusUpdate", unpack(expTable))
+  :Times(#expTable)
+  common.run.wait(5000)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL and HMI", common.start)
+runner.Step("Connect two mobile devices to SDL", common.connectMobDevices, {devices})
+
+runner.Title("Test")
+runner.Step("Register App1 from device 1 - No PTU", registerApp1)
```

Comment:
Propose to update the description to`Register App1 from device 1 - No PTU before ApplicationListUpdateTimeout is expired `

Reply:
@GetmanetsIrina Please find update in 0e0c08b